src/CNGLOB.H

```diff
@@ -84,6 +84,9 @@ typedef long long LLI;		// signed 64 bit integer
 typedef unsigned long long ULLI;
 //typedef int BOOL;	if needed: // 16 or 32 bit Boolean, matches windows.h.
 
+/*---------------------------- Remove DLL Support ---------------------------*/
+#undef SUPPORT_DLLS
```

Comment:
I think we can do something a little more descriptive and specific than `SUPPORT_DLLS`. Maybe `ELIMINATE_XMODULE`?

Reply:
Let's go with `SUPPORT_XMODULE` so we don't have to change all the references to be negative. @chipbarnaby let us know if you have any other concerns with this PR otherwise we'll merge.